SweetsFactory/src/app/shared/account/account.component.css

```diff
@@ -0,0 +1,18 @@
+.disable-click {
+    pointer-events: none;
+    cursor: default;
```

Comment:
Is the class is called .disable-click showing regular mouse pointer is not ok, please change the cursor property to _cursor: not-allowed;_